### PR TITLE
Remove TestObject from the code (EOL for TO) 

### PR DESCRIPTION
**Pull request by eyaly**

### CODE REVIEW

File: assets/locales/es-ES/translation.json [hunk 1]
```diff
@@ -195,7 +195,6 @@
   "elementsCount": "Elements ({{elementCount}})",
   "xCoordinate": "X: {{x}}",
   "yCoordinate": "Y: {{y}}",
-  "TestObject Data Center": "TestObject Data Center",
```
[the remark this answers is not in the document]
Comment (eyaly) [reply assets/locales/es-ES/translation.json]: Done

**Comment by mykola-mokhnach** [comment]
@eyaly Could you please also sign the CLA?

**Comment by eyaly** [comment]
@mykola-mokhnach - I signed the CLA

**Review by jlipps — APPROVED** [review]
thanks!

**Pull request merged**